.travis.yml

```diff
@@ -1,5 +1,7 @@
 language: go
 
 go:
-  - 1.4
-  - tip
\ No newline at end of file
+  - '1.4'
```

Comment:
I left this in for the sake of "no surprises", but in theory 1.4 is long deprecated, so it could also be removed.

Reply:
Thank you for trying to avoid surprises, but it is probably time to remove that.

Reply:
I happily removed it.